pages/style.css

```diff
@@ -132,4 +132,19 @@ p {
 .social a, .social a:visited {
     margin-top: 4px;
     color: rgba(50, 204, 0, 0.4);
-}
\ No newline at end of file
+}
+
+.Dropdown-root {
```

Comment:
Ok for me, but maybe using just native select could have been simpler.

Reply:
difficult to style simple select to look like this, and react-dropdown quite light lib 

<img width="1203" alt="image" src="https://user-images.githubusercontent.com/13865639/155897716-9ba932e4-10fc-4e45-945d-0034edc6af38.png">